joomla/joomla-cms
# [#5570] - Create Article:: Can we change the placeholder message for 'Tag'

Provide possible solution for issue #5570

**Request:** 
Placeholder message for tag field should be "Type or select some options".

**More information:**

The input field of tags which you find at the article manager in the back end has the placeholder message "Select an option". This can lead the user to think that there it is only the possibility to select one of the already given tags. But the actual options you have through selecting the input field are:
1. Select one tag from available list
2. Select multiple tags from available list
3. Type a tag
4. Type multiple tags
5. Combination of selected and typed tags

In order to make the bahavior of this input field clearer the placeholder message should be changed to something like: "Type or select some options" 

**The solution for this issue:**
1. The string used as placeholder is from the global variable "JGLOBAL_SELECT_SOME_OPTIONS" in the "en-GB.ini".
2. In the "formbehavior.php" this variable is used as placeholder when the field is marked as "multiple" in a method named "chosen".
3. Created a new global variable named "JGLOABL_TYPE_OR_SELECT_SOME_OPTIONS" in "en-GB.ini" which contains the string "Type or select some options".
4. Changed the method "chosen" in "formbehavior.php" so if the field is marked as "multiple" the string given for the placeholder message is the created "JGLOABL_TYPE_OR_SELECT_SOME_OPTIONS". 

**How to test:**

-Log into the administrator panel
-Go to article manager
-Create new article
-Check placeholder message

**Expected result after solution:**
The placeholder message of the tag field will now be "Type or select some options."

_Worked as a group on that issue: @icampus, @FPerisa, @kathastaden and @xsability_
